src/db/db/dbPCellDeclaration.h

```diff
@@ -30,6 +30,7 @@
 #include "dbLayout.h"
 #include "tlVariant.h"
 #include "tlObject.h"
+#include "tlOptional.h"
```

Comment:
That's a general style question: Use a specific value of item('s) to decide whether it is used or not, or wrap it in an optional? I prefer using optional because of it's speaking name and readability increasing concept: I'm optional.